CHANGELOG

```diff
@@ -1,6 +1,15 @@
 devel
 -----
 
+* APM-283: Use parallel gather in almost all queries. The only case were we
+  still cannot use parallel gather is when using traversals. All other queries
+  should now be able to parallelize the gather node. This can not only speed up
+  queries quite significantly, but also reduce the memory required by the
+  coordinator for queries like `FOR i IN @n INSERT {} INTO collection`. The
+  serial gather node processes one db server after the other, but since the new
```

Comment:
The standard way of writing "db server" in our documentation seems to be "DB-Server":
```
$ grep -rio "db.*server" ./docs/3.11/ | grep -v "generated" | cut -d ":" -f 2 | sort | uniq -c | sort -n
...
      9 DBSERVER
     25 dbserver
     26 DB Server
     29 DBServer
     85 DB server
    305 DB-Server
```

```suggestion
  serial gather node processes one DB-Server after the other, but since the new
```